clients/hadoop-to-venice-bridge/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -639,6 +649,11 @@ private PushJobSetting getPushJobSetting(String veniceControllerUrl, VenicePrope
     pushJobSettingToReturn.kafkaInputCombinerEnabled = props.getBoolean(KAFKA_INPUT_COMBINER_ENABLED, false);
     pushJobSettingToReturn.suppressEndOfPushMessage = props.getBoolean(SUPPRESS_END_OF_PUSH_MESSAGE, false);
     pushJobSettingToReturn.deferVersionSwap = props.getBoolean(DEFER_VERSION_SWAP, false);
+    pushJobSettingToReturn.repushTtlInHours = props.getLong(REPUSH_TTL_IN_HOURS, UNSET);
```

Comment:
the TTL policy should be read from prop with default RT_WRITE so that user can control the policy.

Reply:
Given we only support one policy, I don't plan to open this config to the user.